cfme/cloud/keypairs.py

```diff
@@ -97,6 +98,8 @@ def is_displayed(self):
     sidebar = View.nested(KeyPairDetailsAccordion)
     entities = View.nested(KeyPairDetailsEntities)
 
+    smart_management = SummaryTable('Smart Management')
```

Comment:
**Required**

Remove this, as the widget is defined in KeyPairDetailsEntities

Reply:
Done